packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -100,6 +102,8 @@ BuildRequires: npm(babel-preset-react) >= 6.24.1
 BuildRequires: npm(babel-preset-react) < 7.0.0
 BuildRequires: npm(identity-obj-proxy) >= 3.0.0
 BuildRequires: npm(identity-obj-proxy) < 4.0.0
+BuildRequires: npm(react-router-dom) >= 4.2.2
```

Comment:
that looks not sorted to me? :)

Reply:
How so? If you do sort it alphabetically I think `i < rea < red` is correct. Do you mean that `<` comes before `>` in ascii because I prefer it this way.

Reply:
Oh wait. It moved from dependencies to devDependencies. That's why it jumped.

I wondered why react-router-dom is not next to react-router-bootstrap anymore